hack/istio-config-1.7.6.yaml

```diff
@@ -5,8 +5,16 @@ metadata:
 spec:
   profile: "demo"
   hub: gcr.io/istio-release
+  meshConfig:
+    # Change the following lines to configure the trust domain of the Istio cluster.
+    trustDomain: foo.bar
+    defaultConfig:
+      proxyMetadata:
+        TRUST_DOMAIN: foo.bar
   values:
     global:
+      # Change the following line to configure the trust domain of the Istio cluster.
+      trustDomain: foo.bar
```

Comment:
This took me a bit of time to figure out, updating all of these fields is required before Istio v1.8.0.